Terminal.Gui/Terminal.Gui.csproj.DotSettings

```diff
@@ -0,0 +1,2 @@
+﻿<wpf:ResourceDictionary xml:space="preserve" xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml" xmlns:s="clr-namespace:System;assembly=mscorlib" xmlns:ss="urn:shemas-jetbrains-com:settings-storage-xaml" xmlns:wpf="http://schemas.microsoft.com/winfx/2006/xaml/presentation">
+	<s:String x:Key="/Default/CodeInspection/CSharpLanguageProject/LanguageLevel/@EntryValue">CSharp120</s:String></wpf:ResourceDictionary>
```

Comment:
What does this do?

Reply:
It's the settings file that is shareable for a given project, for anyone using ReSharper or Rider.

It is what controls a lot of things for Rider and ReSharper, including inspections, formatting, refactoring, and a whole slew of other goodies.

That commit was just the initial addition of the file and only defines the language version for it to consider for analysis.

A key additional change coming soon will add formatting settings that correspond to the project's preferences.